.goreleaser.yaml

```diff
@@ -123,6 +123,12 @@ docker_manifests:
       - "flashbots/suave-geth:latest-amd64"
```

Comment:
Not sure If I have to do something with this.

Reply:
yes, I think this whole block should look like:

```yaml
  {{- if not .IsNightly }}
  - name_template: "flashbots/suave-geth:latest"
    image_templates:
       - "flashbots/suave-geth:latest-amd64"
       - "flashbots/suave-geth:latest-arm64"
  {{ else }}
  - name_template: "flashbots/suave-geth:latest-nightly"
    image_templates:
       - "flashbots/suave-geth:latest-nightly-amd64"
       - "flashbots/suave-geth:latest-nightly-arm64"
  {{- end }}
```